lib/DDG/Spice/Twitter.pm

```diff
@@ -5,16 +5,22 @@ use strict;
 use DDG::Spice;
 
 spice to => 'https://duckduckgo.com/tw.js?user=$1&callback={{callback}}&current=1';
-triggers query => qr/^(?:twitter\s)?@([a-z0-9_]+)$|^twitter\s([a-z0-9_]+)$/i;
+spice proxy_cache_valid => '200 1h';
+
+my $grammar = qr/on|at/;
+my $other = qr/user|account|id/;
+
+triggers query_lc => qr/^(?:twitter\s)?(?:$other\s)?@([a-z0-9_]+)$/i;
+triggers query_lc => qr/^twitter\s(?:$other\s)?([a-z0-9_]+)$/i;
+triggers query_lc => qr/^([a-z0-9_]+)(?:'s)?\s(?:$other\s)?(?:$grammar\s)?(?:twitter)(?:\s$other)?$/i;
+triggers query_lc => qr/^@([a-z0-9_]+)(?:'s)?\s(?:$other\s)?(?:$grammar\s)?twitter?(?:\s$other)?$/i;
```

Comment:
So I think I can leave the handling of `@account twitter account` or `@account's twitter account` type of searches if those are not included in internal changes?